src/main/resources/assets/thaumichorizons/lang/ru_RU.lang

```diff
@@ -223,324 +221,405 @@ tile.ThaumicHorizons_crystal.9.name=Бирюзовый кристалл
-tc.research_name.transductionAmplifier=Узел-Стимулятор
-tc.research_text.transductionAmplifier=Вис, сэр!
-transductionAmplifier1=Без вмешательства извне, узел ауры накапливает вис рывками, вызванными флуктуациями во временном разрыве. Ваш узел-преобразователь может держать узел открытым, но его размер строго ограничен количеством базового вис, который в нем содержится, увеличение объема узла было невозможным до того момента, как вы разработали теорию подпространств. <BR> Используя новое устройство, полученное вами в ходе исследований, вы создали механизм, позволяющий создавать микроскопические подпространственные разломы внутри ядра возбужденного узла. 
-transductionAmplifier2=Подсоединение Узел-Стимулятора к возбужденному узлу и подача редстоун-сигнала увеличит количество санти-вис в узле, как если бы в используемом узле было на 10 вис каждого типа больше. Увеличивая размеры узла, вы подвергаете пространственные нарушения значительным нагрузкам. После присоединения одного стимулятора ничего не случится, но подсоединяя второй, будьте готовы к выбросам межпространственной энергии, третий стимулятор даст огромное количество выбросов энергии заражения. <BR> Совет: не пытайтесь присоединять четвертый. Это приведет к катастрофе!
-tc.research_name.vortexStabilizer=Вихревой Сдерживатель
+tc.research_name.tumorOops=Всё - лишь часть Схемы
+tumorOops1=Вы верили, что постигли всё о своём мире и его содержимом. Глупец.<BR>Всё наконец прояснилось. Древние это не существа в обычном понимании этого слова, не нелепые создания из плоти и крови, вроде Вас. Они - это схема, записанная в звёздах, в потоках и завихрених магии, закодированная в структуре самой жизни.<BR>Но Схема была разбита на части, отделённые друг от друга и рассеянные по всему космосу неким первичным катаклизмом, о природе которого можно только гадать. Она ищет способ воссоздать себя ещё с начала времён, и теперь Вы сыграли роль в её жизненном цикле.
+tumorOops2=Все Ваши исследования, творения, путешествия служили плану Схемы. Изучение законов магии впустило её в Ваш разум. Создание запретных артефактов позволило ей воссоздать себя в физической форме. И, раз уж Вы пережили испытания Внешних земель, значит Схема решила, что Вы собрали достаточную её часть.<BR>Достаточную для создания высшего существа. Следующей стадии её жизненного цикла.<BR>Но Вы ещё жизы. Возможно, ещё не поздно. Возможно Вы сможете лучше понять Схему- возможно Вы ещё можете её остановить-<BR>Или взрастить ещё сильнее. Гораздо сильнее.
+
+tc.research_name.construction=Магическое строительство
+tc.research_text.construction=Не говорите профсоюзу големов
+construction1=TODO
```

Comment:
Отвечу тебе тут сразу за всё
Непереведённые фрагменты и TODO это части, которые сам автор добавил в языковой файл, но так и не реализовал в моде, так что я не стал их переводить просто потому что этот текст в любом случае нигде не используется. Если есть необходимость (либо если кто-то всё-таки добавит эти штуки в мод) - я могу доперевести эти фрагменты
Specially for our english speaking colleagues (I'm too lazy to write the same thing again in English, so I'll use an automatic translator):
I will answer you here at once for everything
The untranslated fragments and TODO are parts that the author himself added to the language file, but never implemented in the mod, so I did not translate them simply because this text is not used anywhere anyway. If you want me to (or if someone will add these things to the mod one day) - I can translate these fragments

Reply:
Аа, ок, если это бесполезный текст, то конечно делать с ним ничего не надо)